src/flask_statsd/__init__.py

```diff
@@ -31,6 +30,26 @@ def setup_middleware(self):
     def statsd_start_timers(self):
         g.request_started_at = time.time()
 
+    def statsd_submit_timers(self, response):
+        def _make_stat_name(req):
+            url_rule = getattr(req.url_rule, 'rule', '')
+            return (
+                url_rule
+                    .lstrip("/")  # drop leading slash
+                    .replace("/", ".")  # use traditional statsd delimiter
+            )
```

Comment:
We need to make it configurable, for example use `metric` as a prefix for these.